common/notification/messages.ts

```diff
@@ -0,0 +1,62 @@
+import { message_translation_language_enum as MessageTranslationLanguage } from '@prisma/client';
+import { getSampleContext } from './notification';
+import { Notification, NotificationMessage, TranslationTemplate } from './types';
+import { prisma } from '../prisma';
+import { TranslationLanguage } from '../entity/MessageTranslation';
+import { NotificationType } from '../entity/Notification';
+import { renderTemplate } from '../../utils/helpers';
+
+export async function getMessageForNotification(
+    notificationId: number,
+    language: TranslationLanguage = TranslationLanguage.DE
+): Promise<NotificationMessage | null> {
+    const notification = await prisma.notification.findUnique({
+        where: { id: notificationId },
+        select: { type: true },
+    });
+    const translation = await prisma.message_translation.findFirst({
+        where: { notificationId: notificationId, language },
+        select: { template: true, navigateTo: true },
+    });
+
+    const { headline, body } = translation.template as any as TranslationTemplate; // @TODO: is it possible to fix any?
```

Comment:
translation can be null no? So translation.template might fail?

Reply:
added some changes